test/unit/polymer-dom.js

```diff
@@ -917,4 +917,21 @@ suite('Polymer.dom non-distributed elements', function() {
     assert.equal(Polymer.dom(document.createElement('div')).getDestinationInsertionPoints().length, 0);
     assert.equal(Polymer.dom(document).getDestinationInsertionPoints().length, 0);
   });
+
+  test('Deep Contains', function() {
```

Comment:
Let's add a couple tests for non-distributed children since we have code that covers this case.